packages/core/src/internal/utils/identity.ts

```diff
@@ -58,8 +58,8 @@ export function getEnumValues(enumeration: any) {
   return Object.values(enumeration);
 }
 
-export function createId() {
-  return `_${Math.random().toString(36).substr(2, 9)}`;
+export function createId(prefix = '_') {
+  return `${prefix}${Math.random().toString(36).substr(2, 9)}`;
```

Comment:
allowing for a prefix gives us the flexibility to use these identifiers to identify types of things if we want them to.